lcviz/plugins/plot_options/plot_options.py

```diff
@@ -31,6 +33,20 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.docs_link = f"https://lcviz.readthedocs.io/en/{self.vdocs}/plugins.html#plot-options"
 
+    def _default_tpf_stretch(
+            self, vmin_percentile=5, vmax_percentile=99, tpf_viewer_reference='image'
+    ):
+        viewer = self.app.get_viewer(tpf_viewer_reference)
+        image = viewer.layers[0].get_image_data()
+        vmin, vmax = np.nanpercentile(
+            image, [vmin_percentile, vmax_percentile]
+        )
+
+        self.viewer_selected = tpf_viewer_reference
+        self.stretch_function_value = 'log'
+        self.stretch_vmin_value = vmin
+        self.stretch_vmax_value = vmax
```

Comment:
can we do this in a separate copy of the plugin to avoid changing the UI itself?

```suggestion
        po = self.new()
        po.viewer_selected = tpf_viewer_reference
        po.stretch_function_value = 'log'
        po.stretch_vmin_value = vmin
        po.stretch_vmax_value = vmax
```